src/walletSdk/index.ts

```diff
@@ -35,19 +36,28 @@ export class Wallet {
 
   constructor(
     stellarConfiguration: StellarConfiguration,
-    applicationConfiguration: ApplicationConfiguration = new ApplicationConfiguration()
+    applicationConfiguration: ApplicationConfiguration = new ApplicationConfiguration(),
+    // Defaults wallet language to "en", this will reflect in all Anchor API calls
+    language: string = "en",
```

Comment:
Is there a difference between "en" and "en-US"?

Reply:
@Ifropc so I left the default value as `en` because the [SEP](https://github.com/stellar/stellar-protocol/blob/master/ecosystem/sep-0024.md#request) mentions it. But, some Anchors (like MGI) may require the wallet to send it using the locale format like `en-US`, because the same language could vary depending on the country. E.g. we could have `en-US`, `en-UK`, `es-AR`, `es-ES`, `pt-BR`, `pt-PT`, etc...

Does that make sense?

Reply:
If you notice on the tests I always used it as `en-US` just for devs to keep in mind that the `lang` parameter could be used with this `language-locale` format

Reply:
Yeah I was just curious if "en" and "en-US" are the same

Reply:
I believe it'll depend more on the Anchor but probably yes :)